fgpyo/sam/__init__.py

```diff
@@ -547,6 +547,40 @@ def length_on_target(self) -> int:
         """Returns the length of the alignment on the target sequence."""
         return sum([elem.length_on_target for elem in self.elements])
 
+    def get_alignment_offsets(self, reverse: bool = False) -> Tuple[int, int]:
+        """
+                Get the starting and ending offset for the alignment based on the CIGAR string.
+
+                Args:
+                    reverse: Whether to count from the end of the read sequence.
+                        Default is False, which counts from the beginning of the read sequence.
+
+                Returns: Tuple[int, int] The start and end of the _aligned part_ of the read.
+                    0-based, open-ended offsets (to the read sequence.)
+                    If 'reverse' is True, the offsets count from the end of the read sequence.
+
+        # TODO: FIGURE OUT WHY the following three lines causes mkdocs to fail
+        # If the Cigar contains no alignment operators that consume sequence bases, or
+        # only clipping operators, the start and end offsets will be the same value (indicating
+        # an empty region).
```

Comment:
**question** What will that value be? Will it always be zero, or the length of the read sequence, or something else?

Reply:
thanks. I expanded the (commented out for now) documentation to clarify.